x-pack/legacy/plugins/siem/server/plugin.ts

```diff
@@ -60,7 +60,7 @@ export class Plugin {
             ],
             read: ['config'],
           },
-          ui: ['show'],
+          ui: ['show', 'crud'],
```

Comment:
Haven't seen this before but I also don't know much about what this `ui` setting does. Going to trust this belongs.

Reply:
FYI https://www.elastic.co/guide/en/kibana/current/development-plugin-feature-registration.html#_ui_capabilities

Reply:
So you do not have to trust!